src/includes/classes/VsUpgrades.php

```diff
@@ -94,6 +94,9 @@ protected function fromLte151107()
                 if (!empty($existing_options['cache_clear_xml_sitemap_patterns']) && strpos($existing_options['cache_clear_xml_sitemap_patterns'], '**') === false) {
                     $this->plugin->options['cache_clear_xml_sitemap_patterns'] = str_replace('*', '**', $existing_options['cache_clear_xml_sitemap_patterns']);
                 }
+                if (!empty($existing_options['exclude_hosts']) && strpos($existing_options['exclude_hosts'], '**') === false) {
```

Comment:
You won't need these lines, because this option did not exist prior; i.e., nothing to upgrade.